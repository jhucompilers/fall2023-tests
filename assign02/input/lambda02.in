function mkaddn(n) {
  function(x) { x + n; };
}

var add1;
var add2;
add1 = mkaddn(1);
add2 = mkaddn(2);

println(add1(1));
println(add2(1));
