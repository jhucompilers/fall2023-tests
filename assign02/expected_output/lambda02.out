2
3
Result: 0
